cmd/terramate/e2etests/cloud/run_cloud_sync_preview_test.go

```diff
@@ -78,12 +87,31 @@ func TestCLIRunWithCloudSyncPreview(t *testing.T) {
 							Cmd:    []string{TerraformTestPath, "plan", "-out=out.tfplan", "-no-color", "-detailed-exitcode"},
 						},
 					},
+					ReviewRequest: &cloud.ReviewRequest{
+						Platform:       "github",
+						Repository:     "terramate.io/terramate-io/dummy-repo.git",
+						CommitSHA:      "6dcb09b5b57875f334f61aebed695e2e4193db5e",
+						Number:         1347,
+						Title:          "Amazing new feature",
+						Description:    "Please pull these awesome changes in!",
+						URL:            "https://github.com/octocat/Hello-World/pull/1347",
+						Labels:         []cloud.Label{{Name: "bug", Color: "f29513", Description: "Something isn't working"}},
+						Status:         "open",
+						ReviewRequired: true,
```

Comment:
This clashes with https://github.com/terramate-io/terramate/pull/1525 . Have you rebase from main?

Reply:
Not yet, will address this and ping you again.